core/metacling/src/TCling.cxx

```diff
@@ -6846,6 +6846,15 @@ void TCling::CallFunc_SetFuncProto(CallFunc_t* func, ClassInfo_t* info, const ch
    f->SetFuncProto(ci, method, funcProto, objectIsConst, offset, mode);
 }
 
+std::string TCling::CallFunc_GetWrapperCode(CallFunc_t *func) const
+{
+   TClingCallFunc *f = (TClingCallFunc *)func;
+   std::string wrapper_name;
+   std::string wrapper;
+   f->print_wrapper(wrapper_name, wrapper);
```

Comment:
Does print_wrapper actually show things on the terminal?  If not, wouldn't it be better to call it get_wrapper_code?